src/routes/_surfaces/workspace/explore/leaderboards/DimensionLeaderboardEntrySet.svelte

```diff
@@ -21,7 +21,7 @@ see more button
   const dispatch = createEventDispatcher();
 </script>
 
-{#each values as { label, value, formattedValue } (label)}
+{#each values as { label, value, __formatted_value } (label)}
```

Comment:
Previously we were only serving `{label, value}` array of objects throughout our codebase. Now with the addition of dimension table the datum objects are of the form `{label, measure1, measure2...}`. A generic key for storing the formatted value won't work. To define these values are internal value and to avoid conflict with measure names have adopted the double underscore variable. So formatting `{label, measure1, measure2..}` returns `{label, __formatted_measure1, __formatted_measure2, ...}`